pornography-hosts

```diff
@@ -15278,3 +15278,6 @@
 0.0.0.0 gotxx.com
 0.0.0.0 mydesi.net
 0.0.0.0 nuefliks.club
+0.0.0.0 desixflix.com
+0.0.0.0 masturbate.com
```

Comment:
Already listed in [porn records](https://mypdns.org/my-privacy-dns/porn-records/-/issues/138) and therfore also clefspar13 project

https://mypdns.org/my-privacy-dns/porn-records/-/commit/42c48224adf9054aa8ee6ddf071dcb79558ae438#a9f5293603129b4fb4ad2abffd25ad984ccf58ad_1368_1368